daemon/lib/source/DobbyManager.cpp

```diff
@@ -2259,6 +2259,8 @@ void DobbyManager::handleContainerTerminate(const ContainerId &id, const std::un
         // Also run any postHalt hooks in RDK plugins
         if (container->config->rdkPlugins().size() > 0)
         {
+            AI_LOG_INFO("Exit status = %d for container = %s", status, id.c_str());
```

Comment:
Do we really need this info message? As there is higher priority message already:
```
0000002349.211270 <T-22130> MIL: < M:Dobby.cpp F:onContainerStopped L:1868 > container 'sleepy'(734) stopped (status 0x0100)
0000002349.211288 <T-22130> NFO: < M:DobbyManager.cpp F:handleContainerTerminate L:2262 > Exit status = 256 for container = sleepy
```
The only difference is that the status in the `milestone` priority is in hex, but we have same info in both messages, so at least for me it is duplicated information.